thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/grouping/filter/AlertGroupFilter.java

```diff
@@ -0,0 +1,23 @@
+package com.linkedin.thirdeye.anomaly.alert.grouping.filter;
+
+import com.linkedin.thirdeye.datalayer.dto.GroupedAnomalyResultsDTO;
+import java.util.Map;
+
+public interface AlertGroupFilter {
```

Comment:
Some javadoc explaining what AlertGroupFilters do, and scenarios it would be applicable, would be helpful